docs/customize-pulsar/customizing-language-recognition.md

```diff
@@ -18,7 +14,4 @@ add this to your configuration file under the `*.core` section:
       ]
 ```
 
-In the example above, `source.coffee` is the language's scope name (see
-[Finding a Language's Scope Name](/customize-pulsar/language-specific-configuration-settings/#finding-a-language's-scope-name) for more
-information) and `foo` is the file extension to match without the period. Adding
-a period to the beginning of either of these will not work.
+In the example above, `source.coffee` is the language's scope name (see [Finding a language’s scope Name](/customize-pulsar/language-specific-configuration-settings/#finding-a-language%E2%80%99s-scope-name) for more information) and `foo` is the file extension to match without the period. Adding a period to the beginning of either of these will not work.
```

Comment:
This is because I use curly quotes like `’` as a matter of habit (and in Markdown files I map `'` to a command that inserts `’` unless I'm in a code block). I think we'd have to change the slug generation logic because I'm not in control of how our Markdown parser generates a slug from a heading name.

If we absolutely had to, I could change all the `’`s in headings and page titles to `'`, but I'd be willing to do a lot of work to avoid that. :)

Reply:
Oh I see, but there's no actual issue with it, was just trying to find the easiest answer. So in that case that likely means leaving this as is, which should be fine, although I may take a look later on and see if we can modify the slug generation logic to make our lives easier